frontend/lib/app_scaffold.dart

```diff
@@ -40,12 +42,14 @@ class _AppScaffoldState extends State<AppScaffoldComponent> {
           context.go(routePath);
         },
         //child: Text(label),
-        title: Text(label, style: TextStyle( color: Theme.of(context).primaryColor )),
+        title: Text(label,
+            style: TextStyle(color: Theme.of(context).primaryColor)),
       ),
     );
   }
 
-  Widget _buildUserButton(BuildContext context, currentUserState, { double width = 100, double fontSize = 13 }) {
+  Widget _buildUserButton(BuildContext context, currentUserState,
```

Comment:
Curious about these style changes - are you doing this manually or is this auto formatting?

Reply:
If we want to change styling it would be nice to align on them and then have a PR to do them all to make PR reviews / git diffs easier.